gemonlinedb/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Don't ignore definition files
+!defs/*.json
+
+# Don't ignore common xsd definitions
+!schema/common.xsd
```

Comment:
Actually, in the RPM definitions,the contents of `xml` are copied to `$RPM_BUILD_ROOT%{_prefix}/share/%{_package}/xml/`, so if you put the `schema` subdirectory there, then the RPM should already contain them